Assets/LobbyController.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LobbyController : MonoBehaviour
+{
+    public Button ButtonPlay;
```

Comment:
write variables according to naming conventions in C# ,
Reference Link - https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/